substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/classinitialization/ClassInitializationFeature.java

```diff
@@ -295,6 +299,33 @@ private void reportMethodInitializationInfo(String path) {
         }
     }
 
+    private static void reportTrackedClassInitializationTraces(String path) {
+        Map<Class<?>, StackTraceElement[]> initializedClasses = ConfigurableClassInitialization.getInitializedClasses();
+        int size = initializedClasses.size();
+        if (size > 0) {
+            // Print to file if required
+            if (Options.PrintClassInitialization.getValue()) {
+                ReportUtils.report(size + " classes are tracked for initialization", path, "tracked_class_initialization", "txt", writer -> {
+                    initializedClasses.forEach((k, v) -> {
+                        writer.println(k.getName());
+                        writer.println("---------------------------------------------");
+                        writer.println(ConfigurableClassInitialization.getTraceString(v));
+                        writer.println();
+                    });
+                });
+            }
+
+            // Print to console
```

Comment:
This might be redundant as we're already printing the report to a file - we should not duplicate it to the standard output here.

Reply:
fixed